cmd/otel-agent/subcommands/run/command.go

```diff
@@ -183,6 +183,6 @@ func runOTelAgentCommand(ctx context.Context, params *subcommands.GlobalParams,
 	return nil
 }
 
-func newForwarderParams(config coreconfig.Component, l log.Component) defaultforwarder.Params {
-	return defaultforwarder.NewParams(config, l)
+func newForwarderParams() defaultforwarder.Params {
+	return defaultforwarder.NewParams()
```

Comment:
Could we inline this call?

`forwarder.Bundle(defaultforwarder.NewParams()),`

Reply:
I kept the existing code. Code updated.